### ISSUE TITLE
Tickets -> Overview -> Readstatus

### ISSUE THREAD

**Issue by Hirbod** [zammad/zammad#911]
50 tickets inside my overview and I don't know which one I've already read. 
Please add a read/unread status (font-weight from bold to normal) and also a filter.

This is unusable like this and I reopen Tickets over and over again

**Comment by babakkazemi** [comment]
+1 I just realized this is a good feature
A good feature would be to make tickets which have a new answer/comment visible somehow (bold, italic, different colour)

**Comment by Hirbod** [comment]
Also a good hint @babakkazemi 
Merci & affarin :)

**Comment by dasvitamin** [comment]
We had the same 'problem' today, at first i tried to setup a workaround by using triggers to set the priority of tickets:

Trigger A: When a ticket is changed and last customer contact was in the last 1 minutes set priority to 3.
Trigger B: When a ticket is changed and last agent contact was in the last 1 minutes set priority to 1.

This seems to work, but obviously we loose the priority functionality that way.


After doing that, i noticed zammad probably wants u to use the notifications to solve this problem,
when properly configured u could have one overview for new tickets (tickets witch have no owner), and one overview for your own tickets witch are open.

So u could just watch the new ticket overview for any new tickets and when one of your own tickets is changed (e.g. a customer send a response) u would know about that because of an notification.

But honestly i would also appreciate any form readstatus, because i think thats what we are used to?

**Comment by martini** [comment]
Question: On what actions should reset the "read status" of a ticket?

By the way, did you recognise the "next in overview" feature?

<img width="624" alt="screen shot 2017-03-30 at 17 02 07" src="https://cloud.githubusercontent.com/assets/110226/24510903/bcf988da-156a-11e7-9968-deacb4f6cd34.png">

**Comment by Hirbod** [comment]
As soon as I've opened the Ticket, it should be marked as read (without having to change the owner status or somewhat). The read-status need to be synced by user, of course, so it would not collide with multiple users. I did recognize the "next" triggeres, but they are unusbale, as the disapear sometimes. Also, when you leave the overview, you won't know where you did stop by. Currently my system is used by two people and n-customers, so sometimes there is no owner or I just want "to read first" an go on that ticket later.

@martini why did you close the ticket?
